### ISSUE TITLE
libuiohook.dylib does not seem to properly unpack into the build dir on mac arm m1

### ISSUE THREAD

**Issue by submarines414** [TolikPylypchuk/SharpHook#26]
Mac OS ventura. building into .NET Maui. Same workflow as is working on windows 10/11, most current version of visual studio, .NET 6/7

**Comment by TolikPylypchuk** [comment]
Thanks for creating an issue on that. I will look into it whenever I get a chance to access a Mac.

**Comment by TolikPylypchuk** [comment]
Hi! So, this bug exists because SharpHook is not supported on Mac Catalyst which means that it cannot be used in MAUI apps on macOS.

Support for Mac Catalyst is tracked in issue #35 so I'm going to close this one since they overlap. If you want to continue the discussion, we can move it to that issue.

I've tried adding Mac Catalyst support to the underlying libuiohook library, but didn't manage to do it, so I created [an issue in the original libuiohook repo](https://github.com/kwhat/libuiohook/issues/168).
